src/afl-fuzz.c

```diff
@@ -383,9 +383,9 @@ static int stricmp(char const *a, char const *b) {
 static void fasan_check_afl_preload(char *afl_preload) {
 
   char   first_preload[PATH_MAX + 1] = {0};
-  char  *separator = strchr(afl_preload, ':');
+  char * separator = strchr(afl_preload, ':');
```

Comment:
What version of clang-format do you have?

Reply:
I used clang-format-10, code-format again with clang-format-14